torchao/dtypes/floatx/floatx_tensor_core_layout.py

```diff
@@ -456,6 +462,53 @@ class FloatxTensorCoreLayout(Layout):
     mbits: int
 
 
+class FloatxTensor(AffineQuantizedTensor):
+    """
+    Floatx quantized tensor subclass which inherits AffineQuantizedTensor class.
```

Comment:
It might be helpful for new users/developers to add a short sentence to the docstring explaining the difference between floatx and float8.